third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js

```diff
@@ -52,7 +52,7 @@ const USER_PROMPT_DIALOG_URL =
     'pcsc_lite_server_clients_management/user-prompt-dialog.html';
 
 const USER_PROMPT_DIALOG_WINDOW_OPTIONS_OVERRIDES = {
-  'innerBounds': {'width': 300}
+  'width': 500
```

Comment:
Why was the constant changed from 300 to 500? Was it intentional?

Reply:
I remember from our offline discussion that 500 was the correct width..and this is actually the width I always got when testing both app and extension mode...but looking back through previous commits "300" seems to be the original width indeed so will create another PR for that

Reply:
Hmm, I don't remember us discussing the need to increase the width... If you can help me recall the reason for increasing the width let's discuss it. Otherwise, yup, let's revert this to use 300 pixels as it was before.